trade-finance-manager-api/src/v1/controllers/deal.submit.controller.js

```diff
@@ -95,8 +95,7 @@ const submitDealAfterUkefIds = async (dealId, dealType, checker) => {
        * Current requirement only allows AIN & MIN deals to be send to ACBS
        */
       const updatedDeal = await api.updateDeal(dealId, updatedDealWithTasks);
-      if (mappedDeal.submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.AIN
-      || mappedDeal.submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.MIN) {
+      if (dealController.dealCanBeSubmittedToACBS(dealId)) {
```

Comment:
Could we just pass in `mappedDeal.submissionType` here, save a DB call in this function?

Reply:
Also would be great to add a test in the actual submit endpoint `deals-submit.api-test.js` to check that `submitACBSIfAllPartiesHaveUrn` is called on submission - it's already stubbed in the test, but nothing checking if it's called

Not a blocker for this PR as it's already missing coverage, nice to have